tests/end2end/features/qutescheme.feature

```diff
@@ -63,6 +63,19 @@ Feature: Special qute:// pages
         And I hint with args "links normal" and follow a
         Then qute://help/quickstart.html should be loaded
 
+    Scenario: Opening a link with qute://help/index.html/..
+        When I open qute://help/index.html/.. without waiting
+        Then qute://help/ should be loaded
+
+    Scenario: Opening a link with qute://help/index.html/../
+        When I open qute://help/index.html/../ without waiting
+        Then qute://help/ should be loaded
+
+    Scenario: Opening a link with qute://help/img/cheatsheet-big.png/..
```

Comment:
This also fails on CI, for the same reason above. I don't think it's necessary to test this separately though, what do you think?

Reply:
Instead of this I'll just try to open the link with qute://help/img, and add a ```When the documentation is up to date``` line at the beginning.